YCloudYUSA/yusaopeny
# [DS-724] Add color font and background color buttons to the CKEditor 5 panel

Original Issue, this PR is going to fix: [DS-724](https://yusa.atlassian.net/browse/DS-724)


Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use 9.x-2.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

1. Log as admin
2. go to /admin/config/content/formats/manage/full_html
3. in the Text editor field please select the **Ckeditor 5**
4. move these buttons to active panels
 
![image](https://github.com/YCloudYUSA/yusaopeny/assets/744406/0690e888-5f68-4c49-83dc-af21d2a3e264)

6. save the text frmat
7. go to edit 
8. verify that you can see these new buttons
9. verify that these buttons works well
![Pasted image 20230529104630](https://github.com/YCloudYUSA/yusaopeny/assets/744406/5a0c8e9c-c440-41ab-a3d3-2930d3855b76)

![Pasted image 20230529104658](https://github.com/YCloudYUSA/yusaopeny/assets/744406/86c25a3d-5440-4800-8c9e-c7c216ce099e)

## General checks
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/><img src="https://raw.githubusercontent.com/ymcatwincities/openy/9.x-2.x/.github/assets/steps-for-review.png" width="200" alt="Steps for review"/>
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/><img src="https://raw.githubusercontent.com/ymcatwincities/openy/9.x-2.x/.github/assets/drupalorg-email.png" width="200" alt="drupal.org email"/>
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!
